### PR TITLE
Added CounterSpell Wolverhampton

### PR DESCRIPTION
**Pull request by tobilobasalawu**
Added  CounterSpell Wolverhampton Website 🎉.

**Changes**
- I made a copy of the example components, made the required changes, and renamed it Wolverhampton.
- I made a copy of the example-city.js file, made required changes, and renamed it to Wolverhampton

### CODE REVIEW

File: components/Schedule.js
Comment (samdev-7) [inline components/Schedule.js:1]: Seems like you edited the wrong file. Please revert these changes.

File: package-lock.json
Comment (samdev-7) [inline package-lock.json:1]: Please don't edit this file. Thanks!

File: package.json
Comment (samdev-7) [inline package.json:1]: Please don't edit this file. Thanks!

File: yarn.lock
Comment (samdev-7) [inline yarn.lock:1]: Please don't edit this file. Thanks!

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/wolverhampton**](https://counterspell-jpppm7bll.hackclub.dev/wolverhampton) |
|:-:|
|<a href="https://counterspell-jpppm7bll.hackclub.dev/wolverhampton">Screenshot of /wolverhampton</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/27CFpCSL84Ylszp2A-MRfIDytEah0lLpqq6cGkPYGquPk4MgctAV3B1mUahlQj63ILO8NvpovDKtBzoqPxtjsQ.png">(view full size)</a> |







Commit aac52964767cc7030aeddca5286d9806daa7a26b (https://counterspell-jpppm7bll.hackclub.dev).

**Comment by samdev-7** [comment]
A few more changes and it'll be good to go!

File: components/Schedule.js
Comment (samdev-7) [reply components/Schedule.js:1]: Please revert the changes on this file, thanks!

File: package-lock.json
Comment (samdev-7) [inline package-lock.json:1]: Still the same issue, please don't create unneeded file. Thanks!

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Pull request merged**